### DISCUSSION TITLE
Any chance someone has the get_signed function as a javascript function?

### DISCUSSION THREAD

**Discussion by Abcmsaj** [DIGITALCRIMINALS/OnlyFans#599] [Q&A]
Just got my own project but struggling to make any API requests because of the sign/time headers - wondering if anyone has converted the func from py > js?

Thank you

**Comment by OFfriend** [comment]
pure browser javascript or nodejs?

**Reply by Shashank003** [reply]
For some reason, it's logging me out even when I'm generating the correct header value( cross checked with python).
